### PR TITLE
Feature/#31

### PR DESCRIPTION
**Pull request by jonghyunchoe**
- Added image field to Post
- Uploaded images are saved to /media folder with Year/Month/Day recorded

### CODE REVIEW

**Review by mieseung — APPROVED** [review]

File: clone_linkedin/post/models.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 class Post(models.Model):
     title = models.CharField(max_length=50, blank=True)
     content = models.TextField()
+    image = models.ImageField(upload_to="%Y/%m/%d")
```
Comment (gina0605) [inline clone_linkedin/post/models.py]: `blank=True` 안해줘도 되나요?

**Review by gina0605 — COMMENTED** [review]
수고하셨어요~!

File: clone_linkedin/post/models.py (hunk 1 above)
Comment (mieseung) [reply clone_linkedin/post/models.py]: 아 그렇네요.. blank=True, null=True 중에 하나 해줘야할 것 같아요

**Review by gina0605 — COMMENTED** [review]
lgtm 👍

**Pull request merged**